src/components/FavoritesBtn.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import api from '../service/api';
+
+import '../styles/FavoritesBtn.scss';
+
+type FavoritesBtnProps = {
+  movieId: number;
+};
+
+const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
+  const [isFavorite, setisFavorite] = useState(api.isIdInFavorites(movieId));
+
+  const heartUnfilledIcon = '/images/favoriteBtn/heartFilled.svg';
+  const heartFilledIcon = '/images/favoriteBtn/heartUnfilled.svg';
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>): void => {
+    event.preventDefault();
+    isFavorite ? api.setFavoritesId(movieId) : api.deleteFavoritsId(movieId);
+    setisFavorite(api.isIdInFavorites(movieId));
+  };
+
+  return (
+    <a href="#!" className="favorite-btn" onClick={handleClick}>
```

Comment:
Is using of the `<a>` here needed?

Reply:
Here, I wanted to be more specific semantically, that's why decided to wrap an Icon in <a> tag